### ISSUE TITLE
Agario bots are alive ?

### ISSUE THREAD

**Issue by sejo153** [davidmann4/ogar-feeder-bot#609]
Please do not ask for proxies
Please do not talk about cheating on public servers.
Please do not talk about how this won't be fixed.

**Comment by David1ali12** [comment]
Sejo there is no way for agar bots again because agario is using captcha 
there some bots working for 5-10 mins and they get banned 
captcha :/
 you can play alis.io instead it has bots mode

**Comment by systemeg55** [comment]
yep agario bots are alive but they will get disconnected after 15 minutes because of the captcha

**Comment by sejo153** [comment]
Oh damn then :D

**Comment by NuclearC** [comment]
ugh actually they dont disconnect i could use them for more than an hour

**Comment by sejo153** [comment]
@NuclearC  Share with us 15 mins is still 15mins :'D
